packages/main/src/plugin/updater.spec.ts

```diff
@@ -373,10 +373,12 @@ describe('expect update command to depends on context', async () => {
 
     vi.mocked(autoUpdater.checkForUpdates).mockResolvedValue({
       updateInfo: {
-        version: '@debug-next',
```

Comment:
What is happening when we have non-semvers compatible as version ? Such as @debug-next what is the outcome ? False I guess ?

> might not be relevant, but curious if the string is malformed

Reply:
@axel7083 following Florent's comment and based on the most recent commit, if either one of the versions (current or next) is not a valid semver, it will still update `nextVersion` value